src/http/response.ts

```diff
@@ -1,5 +1,6 @@
 import Drash from "../../mod.ts";
 import { STATUS_TEXT, Status } from "../../deps.ts";
+import { setCookie, delCookie, Cookie } from "../../deps.ts"
```

Comment:
since `deps.ts` is already being imported on line 2, you can add `setCookie`, `delCookie`, `Cookie` to line 2. just make sure you stack the list vertically and put it in alphabetical order